src/redux/categories/categories.js

```diff
@@ -0,0 +1,21 @@
+// Action
+const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
+
+// Initial State
+const initialState = { categories: [] };
+
+// Action creator
+export const checkStatus = () => ({ type: CHECK_STATUS });
+
+// Reducer
+const categoriesReducer = (state = initialState, action) => {
+  if (action.type === CHECK_STATUS) {
+    return {
+      ...state,
+      categories: [...state.categories, 'Under Construction'],
+    };
+  }
+  return state;
+};
```

Comment:
- [OPTIONAL] I would advise you to return the text 'Under Construction' when the action that is triggered is `CHECK_STATUS`. That way, it is easier for you to access the state. The way you have it, you would have to write a lot of JavaScript methods to access the state for categories when you want to use it at the front end. 👍 💻

Reply:
@Gambit142 thank you for the suggestion. Does that mean I should return just a text as my state? Any suggestions